composer.json

```diff
@@ -13,6 +13,18 @@
         {
             "type": "composer",
             "url": "https://asset-packagist.org"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "antennaio/jquery-bar-rating",
+                "version": "1.2.2",
+                "type": "drupal-library",
+                "dist": {
+                    "type": "zip",
+                    "url": "https://github.com/antennaio/jquery-bar-rating/archive/v1.2.2.zip"
+                }
+            }
```

Comment:
Dev version of votingapi_widget has a problem with composer installation, needs to add the package as a separate repository. @see https://www.drupal.org/project/votingapi_widgets/issues/2954719